frontend/mgramseva/lib/screeens/connection_results/search_connection.dart

```diff
@@ -148,32 +148,38 @@ class _SearchConsumerConnectionState extends State<SearchConsumerConnection> {
                                       Text(
                                           '\n${ApplicationLocalizations.of(context).translate(i18.common.OR)}',
                                           textAlign: TextAlign.center),
-                                      BuildTextField(
-                                        i18.searchWaterConnection
-                                            .OLD_CONNECTION_ID,
-                                        searchConnectionProvider
-                                            .searchconnection
-                                            .oldConnectionCtrl,
-                                        isDisabled: searchConnectionProvider
-                                            .searchconnection.controllers[2],
-                                        onChange: (value) =>
-                                            searchConnectionProvider
-                                                .getdetails(value, 2),
-                                        hint: ApplicationLocalizations.of(
-                                                context)
-                                            .translate(i18
-                                                .searchWaterConnection
-                                                .OLD_CONNECTION_HINT),
-                                        key: Keys.searchConnection.SEARCH_OLD_ID_KEY,
-                                          onSubmit: (value){
-                                            searchConnectionProvider.validatesearchConnectionDetails(
-                                                context, widget.arguments, (searchConnectionProvider.searchconnection.controllers[1] == false)
-                                                ? true : false);
-                                          }
+                                      Visibility(
```

Comment:
Understood that concept but as of know we are doing some changes and we are not required that component know in future we need to show them to user so instead of remove that elements I used visibility so, in future if need then we can show them